docs/admin-manual/setup/install.md

```diff
@@ -24,7 +24,9 @@ You can install LibreTime using the one of the following methods:
 #### Minimum system requirements
 
 - One of the following Linux distributions
-  - Ubuntu [current LTS](https://wiki.ubuntu.com/Releases)
+  - Ubuntu [current LTS](https://wiki.ubuntu.com/Releases). Note Ubuntu 22.04
+    LTS is not yet supported. Work is being tracked in
```

Comment:
Maybe make 'Ubuntu 22.04 is not supported' a link to the github issue using `[]()` ? And remove the 'work is being tracked in LINK'